packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -121,6 +149,47 @@ export function connectAugur(
     getState: () => AppState
   ) => {
     const { modal, loginAccount } = getState();
+    const windowApp = windowRef as WindowApp;
+
+    const loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
+    const loggedInAccountType = windowApp.localStorage.getItem('loggedInAccountType');
+
+    // Preload Account
+    const preloadAccount = (accountType) => {
+      const accountObject = {
+        address: loggedInAccount,
+        mixedCaseAddress: toChecksumAddress(loggedInAccount),
+        meta: {
+          address: loggedInAccount,
+          signer: null,
+          email: null,
+          profileImage: null,
+          openWallet: null,
+          accountType,
+          isWeb3: true,
+          preloaded: true,
+        },
+      };
+      dispatch(updateAuthStatus(IS_LOGGED, true))
```

Comment:
Can’t set logged in here. The sdk needs to be synced in order to be officially “logged in”.  See how logged in is set in log-handler when user synced event comes in

Reply:
Can just remove setting logged in true here. Should all still work. If the sdk is told about the account

Reply:
Didn't seen any issues during testing, but I think you are right here.

Yeah, in this case, I'll have to create another authStatus I think, something like PRELOADED, to control showing things like Account details and avatar,  while we wait for syncing to complete

Does that make sense?

Reply:
> Can just remove setting logged in true here. Should all still work. If the sdk is told about the account

If I do this, currently, the user won't appear logged in till the SDK is loaded + markets. Which is a long delay vs being able to show that right on page load.